Dockerfile

```diff
@@ -1,13 +1,10 @@
-FROM continuumio/anaconda
+FROM jupyter/scipy-notebook
 
-# enable utf-8 locale
-ENV PYTHONUNBUFFERED 1
+COPY environment.yml ./
+USER root
+RUN chown jovyan:users environment.yml
+USER jovyan
 
-RUN conda install -y scikit-learn jupyter
+RUN conda env update --prune
```

Comment:
Better to combine these to one statement using `&&`

Reply:
You're right. There's not much benefit to keeping them separate